src/ProgressPane.jsx

```diff
@@ -148,7 +148,7 @@ function DataView({ data, captionPortal, nonce, color }) {
             </button>
         </div>
 
-        {data.length && selectedEntry && captionPortal.current && createPortal(<EntryInfo entry={selectedEntry} isLatest={selectedEntry.commit === latest.commit}/>, captionPortal.current)}
+        {(data.length && selectedEntry && captionPortal.current && createPortal(<EntryInfo entry={selectedEntry} isLatest={selectedEntry.commit === latest.commit}/>, captionPortal.current) || null)}
```

Comment:
this would resolve to 0 and leave an actual zero as an artifact for a few milliseconds when switching between progress charts.